common/src/main/java/com/thekillerbunny/goofyplugin/GoofyPlugin.java

```diff
@@ -22,7 +22,7 @@
 @FiguraAPIPlugin
 @LuaWhitelist
 public class GoofyPlugin implements FiguraAPI {
-    public static final String PLUGIN_ID = "goofyfiguraplugin";
+    public static final String PLUGIN_ID = "GoofyPlugin";
     public static final Logger LOGGER = LoggerFactory.getLogger(PLUGIN_ID);
```

Comment:
don't capitalize IDs
```suggestion
    public static final String PLUGIN_ID = "goofyplugin";
    public static final Logger LOGGER = LoggerFactory.getLogger("GoofyPlugin");
```